dev-tools/omdb/tests/test_all_output.rs

```diff
@@ -326,9 +436,20 @@ async fn do_run_extra<F>(
     write!(output, "termination: {:?}\n", exit_status).unwrap();
     write!(output, "---------------------------------------------\n").unwrap();
     write!(output, "stdout:\n").unwrap();
-    output.push_str(&redact_extra(&stdout_text, extra_redactions));
+    if redactions {
+        output.push_str(&redact_extra(&stdout_text, extra_redactions.unwrap()));
+    } else {
+        output.push_str(&stdout_text);
+    }
```

Comment:
Following on the comment above, I'd change this to either an `if let` or an `Option::map` on `extra_redactions`.